crabs/detection_tracking/detection_utils.py

```diff
@@ -186,5 +190,127 @@ def draw_detection(
                         (0, 0, 255),
                         label_text,
                     )
-
     return image_with_boxes
+
+
+def load_ground_truth(ground_truth_paths: str) -> dict:
+    """
+    Load ground truth annotations from JSON files.
+
+    Parameters
+    ----------
+    ground_truth_paths : str
+        The directory containing the ground truth JSON files.
+
+    Returns
+    -------
+    dict
+        A dictionary mapping object IDs to their corresponding ground truth annotations.
+    """
+    ground_truths = {}
+    for filename in os.listdir(ground_truth_paths):
+        if filename.endswith(".json"):
+            object_id = filename.split("_")[-1].split(".")[
```

Comment:
yes, this is when I load Nanci's ground truth json files. because every crab id is in separate files, I take the filename as the crab_id. Should we remove this for now?

Reply:
yeah let's remove it for now to avoid confusion, we are generating new ground truth anyways right?